### PR TITLE
fix(docs): indent code blocks correctly in ordered lists

### PR DESCRIPTION
**Pull request by jharrell**
Resolves WEB-415

### CODE REVIEW

**Review by janpio — APPROVED** [review]

File: content/200-orm/500-reference/375-supported-databases.mdx
Comment (janpio) [inline content/200-orm/500-reference/375-supported-databases.mdx:1]: Unrelated change?

Comment (jharrell) [reply content/200-orm/500-reference/375-supported-databases.mdx:1]: Yeah, sorry. Came up in a search of open docs issues. Thought I'd throw it in.
